<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QWidget" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#content 
{
	background-color: #26374C;
	border: 1px solid #344457;
	border-bottom: 1px solid transparent;
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 0px;
	margin-left: 0px;
	margin-right: 0px;
}

QLineEdit#title_bar_search
{
	border: 0px solid transparent;
	border-top-left-radius: 4px;
	border-bottom-left-radius: 4px;
	border-top-right-radius: 0px;
	border-bottom-right-radius: 0px;
	padding-top: 3px;
	padding-bottom: 3px;
	padding-left: 2px;
	padding-right: 2px;
	background: rgba(44, 64, 90, 255);
	color: white;
	selection-background-color: #0190EA;
}

QLineEdit::focus
{
	border: 1px solid #0190EA;
	border-top-left-radius: 4px;
	border-bottom-left-radius: 4px;
	border-top-right-radius: 0px;
	border-bottom-right-radius: 0px;
	padding-top: 2px;
	padding-bottom: 2px;
	padding-left: 5px;
	padding-right: 3px;
	background: rgba(28, 41, 59, 128);
	color: white;
	selection-background-color: #0190EA;
}

QLineEdit#title_bar_search::hover
{
	border: 0px solid transparent;
	border-top-left-radius: 4px;
	border-bottom-left-radius: 4px;
	border-top-right-radius: 0px;
	border-bottom-right-radius: 0px;
	padding-top: 3px;
	padding-bottom: 3px;
	padding-left: 2px;
	padding-right: 2px;
	background: rgba(44, 64, 90, 255);
	color: white;
	selection-background-color: #0190EA;
}

QComboBox 
{
	border: 0px solid transparent;
	border-left: 1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(44, 64, 90, 255), stop:0.24 rgba(44, 64, 90, 255), stop:0.25 #75879B, stop:0.75 #75879B, stop:0.76 rgba(44, 64, 90, 255), stop:1 rgba(44, 64, 90, 255));
	border-top-right-radius: 4px;
	border-bottom-right-radius: 4px;
	border-top-left-radius: 0px;
	border-bottom-left-radius: 0px;
	padding-left: 9px;
	padding-right: 3px;
	padding-top: 3px;
	padding-bottom: 3px;
	background: rgba(44, 64, 90, 255);
	color: rgba(255, 255, 255, 200);
}

QComboBox::hover
{
	border: 0px solid transparent;
	border-left: 1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(44, 64, 90, 255), stop:0.24 rgba(44, 64, 90, 255), stop:0.25 #75879B, stop:0.75 #75879B, stop:0.76 rgba(44, 64, 90, 255), stop:1 rgba(44, 64, 90, 255));
	border-top-right-radius: 4px;
	border-bottom-right-radius: 4px;
	border-top-left-radius: 0px;
	border-bottom-left-radius: 0px;
	padding-left: 9px;
	padding-right: 5px;
	padding-top: 3px;
	padding-bottom: 3px;
	background: rgba(44, 64, 90, 255);
	color: rgba(255, 255, 255, 200);
}

QComboBox::focus
{
	border: 0px solid #0190EA;
	border-left: 1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(44, 64, 90, 255), stop:0.24 rgba(44, 64, 90, 255), stop:0.25 #75879B, stop:0.75 #75879B, stop:0.76 rgba(44, 64, 90, 255), stop:1 rgba(44, 64, 90, 255));
	border-top-right-radius: 4px;
	border-bottom-right-radius: 4px;
	border-top-left-radius: 0px;
	border-bottom-left-radius: 0px;
	padding-left: 9px;
	padding-right: 5px;
	padding-top: 3px;
	padding-bottom: 3px;
	background: rgba(44, 64, 90, 255);
	color: rgba(255, 255, 255, 200);
}

QComboBox::drop-down
{
	border: 0px;
	background-color: transparent;
}

QComboBox::down-arrow
{
	image: url(:/images/images/expand_more-white-18dp_opacity.svg);
	width: 18px;
	height: 18px;
	padding-right: 15px;
	padding-top: 0px;
}


QComboBox::down-arrow:on
{
	image: url(:/images/images/expand_less-white-18dp_opacity.svg);
	width: 18px;
	height: 18px;
	padding-right: 15px;
	padding-top: 0px;
}

QComboBox QAbstractItemView
{
	background: #212F41;
	color: white;
	border-top-style: inset;
	border-width: 0px;
	border-radius: 3px;
	border-color: #0190EA;
	selection-background-color: #0190EA;
	padding-top: 5px;
	padding-bottom: 5px;
	padding-left: 2px;
	padding-right: 2px;
	outline: none;
	margin-top: 5px;
}

QComboBox QAbstractItemView::item
{
	min-height: 35px;
}

QComboBox QScollBar:vertical
{
	border: 0px;
	background: transparent;
	width: 10px;
	margin: 1px 1 1px 0;
}

QComboBox QScollBar::handle:vertical
{
	background: #0190EA;
	min-height: 20px;
	border-radius: 0px;
}

QToolButton
{
	background-color: #26374C;
	border-top: 0px solid #344457;
	border-right: 0px solid #344457;
	border-top-left-radius: 0px;
	border-top-right-radius: 0px;
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 0px;
	margin-left: 0px;
	margin-right: 0px;
	padding-bottom: 0px;
	padding-left: 0px;
	padding-top: 0px;
}

QLabel#title_bar_title 
{
	background-color: #26374C;
	border: 0px solid transparent;
	border-top-left-radius: 4px;
	border-top-right-radius: 0px;
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 0px;
	margin-left: 0px;
	margin-right: 0px;
	padding-bottom: 10px;
	padding-left: 0px;
	padding-top: 10px;
	color: white;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="content" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="title_bar_title">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="LineEditWithIcon" name="title_bar_search">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>Search...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="search_field">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Name</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Amount</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Date</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QToolButton" name="add">
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/resources.qrc">
          <normaloff>:/images/images/add_circle_outline-white-24dp.svg</normaloff>:/images/images/add_circle_outline-white-24dp.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditWithIcon</class>
   <extends>QLineEdit</extends>
   <header>view/widgets/line_edit_with_icon.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
